src/components/Table/Table.scss

```diff
@@ -50,6 +50,8 @@
     }
 
     &__cell {
+        position: relative;
```

Comment:
We can now move these styles to the `.checkbox_cell` class

Reply:
[moved](https://github.com/gravity-ui/uikit/pull/654/commits/2861dfe4d12e6258b3abfa5d6d9a233a14e1dc5a)